cumulusci/core/config/project_config.py

```diff
@@ -86,15 +88,15 @@ def _load_config(self):
 
         # Load the project's yaml config file
         with open(self.config_project_path, "r") as f_config:
-            project_config = yaml.safe_load(f_config)
+            project_config = cci_safe_load(f_config)
 
         if project_config:
             self.config_project.update(project_config)
 
         # Load the local project yaml config file if it exists
         if self.config_project_local_path:
             with open(self.config_project_local_path, "r") as f_local_config:
-                local_config = yaml.safe_load(f_local_config)
+                local_config = cci_safe_load(f_local_config)
```

Comment:
There are two other calls to yaml.safe_load in the project_config module that should probably also be adjusted.

Reply:
I had done that but then undid it on the premise that I should make this kind-of-rushed change as low-impact as possible, especially given that there is no reason to think that those obscure CCI.yml files would ever end up with an NBSP in them.

Once the real CCI.yml parser is integrated, all CCI.yml files will be treated the same way, I agree.

If you feel strongly it should be fairly low-risk to change them all, though.

Reply:
I disagree that there's no risk of them ending up with an NBSP, but I'm fine holding off since we anticipate introducing a more robust and consistent validation soon.